download_asfserial_rsmas.py

```diff
@@ -2,10 +2,32 @@
 import os
 import sys
 import subprocess
+import logging
+import argparse
+import datetime
+from rsmas_logging import rsmas_logger
 
 sys.path.insert(0, os.getenv('SSARAHOME'))
 import password_config as password
 
+logfile_name = os.getenv('OPERATIONS') + '/LOGS/asfserial_rsmas.log'
+logger = rsmas_logger(file=logfile_name)
+
+inps = None
```

Comment:
I'm wary of this global variable. Can it be refactored into a local variable?

Reply:
If we do this, then we have to pass it around to all of the functions. This isn't a class so their aren't really "local variables". I prefer to keep it global rather than have to pass inps as a parameter to every function call.

Reply:
I would argue that explicitly passing `inps` to functions would make clear what those functions rely on. It could even be more clear if you only pass `inps.template` to `command_line_parse()` instead of all of `inps`. 

Besides the "_it doesn't feel right_" argument, I think the danger could arise in the case (as in this one) where `inps` ends up in an unknown (or not initiated) state and it's unclear where in the code this happens. 

Also, although this is just a script being run from the command line right now, I could see the use case arising where it is imported and run from another function. Encapsulating `inps` in that case and not exposing it seems like it could be useful.

Reply:
Not personally a fan of passing that parameter around a bunch. Seems to just complicate the argument list for otherwise simple functions, especially when its used in a minimal number of occasions across multiple functions.

But I can see where making an instance variable would be useful if we start accessing it from different scripts. However, I don't think that will ever happen, as each script should have there own inps variables holding the command line argument values for that script, and not care about the inps of other scripts.

Reply:
I would argue that it would increase readability if function inputs are declared rather than implicit.